NEWS.md

```diff
@@ -1,3 +1,12 @@
+# admiralophtha 1.1.0
+
+## Updates to Functions
+
+- The following function arguments are entering the next phase of the deprecation process (#223): 
```

Comment:
Sorry yuki, i missed this on the first pass - can you please specify they are arguments of `derive_var_afeye()`? thanks